tools/nstar/src/main.rs

```diff
@@ -199,7 +199,7 @@ async fn command_to_request<T: AsyncRead + AsyncWrite + Unpin>(
 ) -> Result<Request> {
     match command {
         Subcommand::Ident => Ok(Request::Ident),
-        Subcommand::Containers => Ok(Request::Containers),
+        Subcommand::Containers => Ok(Request::InspectAll),
```

Comment:
This is inconsistent. The requests are `Inspect` and `InspectAll`. On the cli there's `inspect` introduced but `containers` stays as it is. On top there's an alias `ls` to `containers`.

I don't really understand why `inspect` is a better match than `containers`. The plural is clear for 'all'. Semantically a "request all containers state" or "request a single container state" feels smoother that. "request a inspection"...

Reply:
no problem, will change it. I personally think inspect is a better choice as it "inspects" the container a container while on the other hand at a first glance, it is not clear what a `container` command does. It is also homologous to docker's inspect command and even `sextant` has an inspect command to inspect npks :wink: